### DISCUSSION TITLE
How to make getting started with Apra Pipes much simpler

### DISCUSSION THREAD

**Discussion by kumaakh** [Apra-Labs/ApraPipes#60] [Ideas]
1. Provide a downloadable public Virtual Machine for linux dev
2. Provide a public SDCard Image for Jetson Nano
3. Provide a public AGX machine for interested parties to try the framework.

@joiskash @omkarsm @saichaitanyaeruvaram @dsiddharth2 please add more ideas here

**Comment by mraduldubey** [comment]
https://github.com/Apra-Labs/ApraPipes/issues/233
